src/Symfony/Component/Debug/Debug.php

```diff
@@ -42,7 +42,7 @@ public static function enable($errorReportingLevel = null, $displayErrors = true
         if (null !== $errorReportingLevel) {
             error_reporting($errorReportingLevel);
         } else {
-            error_reporting(-1);
+            error_reporting(E_ALL);
```

Comment:
This might become an issue in the future when new constants are introduced in future PHP versions (similar to the introduction of the `E_STRICT` constant).